modules/ROOT/pages/6.3-release-notes.adoc

```diff
@@ -599,6 +599,27 @@ In previous versions of {productname}, when running on Firefox and after pressin
 
 This error has been corrected in {productname} 6.3. When running {productname} 6.3 on Firefox and creating a merge tag using `mceInsertContent`, pressing Enter or Return to generate the desired merge tag moves the insertion point to the end of said newly-created merge tag, as expected.
 
+=== The `autoresize` plugin caused the editor area to infinitely resize when `content_css` was set to `document`.
+
+The `document` skin’s CSS aligns the bottom of the content area to the bottom of the editor, but does not set anything for `autoresize_bottom_margin`. As a consequence the {productname} editor takes the default value for this attribute: 50.
+
+Simulaneously, everytime the `resize` function calculated the editor size it added this `autoresize_bottom_margin` value.
+
+Since, however, the aligning CSS sets the bottom of the content area to a value incrementally less than the `autoresize_bottom_margin` value, the `resize` function went into a loop.
```

Comment:
Since, however, the aligning CSS sets the bottom of the content area to a value incrementally less than the `autoresize_bottom_margin` value, the `resize` function went into a loop